hilll2-HW03.py

```diff
@@ -0,0 +1,123 @@
+#TODO Do not edit this file directly! Instead, create a new file called
+#        hw03_username.py and copy this code into it!
+
+#################################################################################
+# Author: Langston Hill
+# Username: hilll2
+#
+# Assignment: Homework 3: Sea of Stars
+# Purpose: To learn the ins and outs of git, along with implementing RGB values into code
+# Google Doc Link: https://docs.google.com/document/d/11EDVFiCCLZRo3eQhymX9W1aXn25xKEMBGwPpwzYjYQI/edit?usp=sharing
+#
+#################################################################################
+# Acknowledgements: https://colorspire.com/rgb-color-wheel/
+# https://docs.python.org/3/library/turtle.html
+#
+#
+#################################################################################
+
+
+import turtle
+import random
+def draw_star(john):
```

Comment:
Pay attention to the gray squigglies, they are style guidelines. This one says you should include two blank lines before all function definitions.